pkgs/development/libraries/xsimd/default.nix

```diff
@@ -1,35 +1,13 @@
 { lib, stdenv, fetchFromGitHub, cmake, gtest }:
-let
-  version = "7.5.0";
-
-  darwin_src = fetchFromGitHub {
-    owner = "xtensor-stack";
-    repo = "xsimd";
-    rev = version;
-    sha256 = "eGAdRSYhf7rbFdm8g1Tz1ZtSVu44yjH/loewblhv9Vs=";
-    # Avoid requiring apple_sdk. We're doing this here instead of in the patchPhase
-    # because this source is directly used in arrow-cpp.
-    # pyconfig.h defines _GNU_SOURCE to 1, so we need to stamp that out too.
-    # Upstream PR with a better fix: https://github.com/xtensor-stack/xsimd/pull/463
-    postFetch = ''
-      mkdir $out
-      tar -xf $downloadedFile --directory=$out --strip-components=1
-      substituteInPlace $out/include/xsimd/types/xsimd_scalar.hpp \
-        --replace 'defined(__APPLE__)' 0 \
-        --replace 'defined(_GNU_SOURCE)' 0
-    '';
```

Comment:
The linked PR has been fixed and I don't see `defined(__APPLE__)` or `defined(_GNU_SOURCE)` in the new version of xsimd source code, so I guess this can be removed now? I don't have a apple machine to test it

Reply:
I don't have one either, but looking at the upstream changes I think it's pretty safe to remove this.